README.md

```diff
@@ -51,18 +53,26 @@ conda install -c conda-forge jupyterlab
 
 #### pip
 
+pip is a package management system for installing and updating Python packages. pip comes with Python, so you get pip simply by installing Python.  On Ubuntu and Fedora Linux, you can simply use your package manager to install the `python3-pip` package.  [*The Hitchhiker's Guide to Python* provides some guidance on how to install Python on your system if it isn't already](https://docs.python-guide.org/starting/installation/); you can also [install Python directly from python.org](https://www.python.org/getit/).  You might want to [upgrade pip](https://pip.pypa.io/en/stable/installing/) before using it to install other programs.
+
+JupyterLab is implemented in Python, and as of 2018-07 JupyterLab can run on either Python2 or Python3.  However, it's recommended that you install JupyterLab on Python3, even when running Python2 code.  Thus, it's recommended that you install Python3, and then use `pip3` to install JupyterLab (`pip3` is the version of `pip` that always uses Python3; in many cases the command `pip` will use Python2, and that is not recommended).
```

Comment:
"as of July 2018" - I think that date is more understandable.

As to the name pip3, I agree that the most universal would be `python3 -m pip`, since the name `python3` is encoded in a PEP: https://www.python.org/dev/peps/pep-0394/.

For *Linux* users, the python docs do mention pip3: https://packaging.python.org/tutorials/installing-packages/?highlight=pip3#ensure-you-can-run-python-from-the-command-line .

Reply:
What should windows users do? Does the stock python for windows have python3? pip3?

Reply:
I just checked on Windows 10 with the 3.6.6 installer from `python.org`, it does not include a `python3` binary.

Reply:
Perhaps the advice should be `python3 -m pip` on Unix-like systems and `python -m pip` on Windows systems.  Whichever we choose for the command examples, it will be awkward for the users on the opposing system since they can't copy and paste.